client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -144,7 +152,13 @@ class CustomContentTypes extends Component {
 				'you can display them using the shortcode [testimonials].',
 			{
 				components: {
-					link: <a href={ localizeUrl( 'https://wordpress.com/support/testimonials/' ) } />,
+					link: (
+						<a
+							onClick={ this.onSupportLinkClick }
+							href={ localizeUrl( 'https://wordpress.com/support/testimonials/' ) }
+							post-id={ 97757 }
```

Comment:
This should work, but I wanted to note that embedding data is a little less common in wp-calypso since we usually have access to app state/know this ahead of time. For example in `mapDispatchToProps`, we might make a more specific action `openTestimonialsSupport`. 

```jsx
const openTestimonialsSupport = e => {
        e.preventDefault();
		openSupportArticleDialog( {
			postId: 97757,
			postUrl: localizeUrl( 'https://wordpress.com/support/testimonials/' ),
		} );
};
const openPortfoliosSupport = e => {
        e.preventDefault();
		openSupportArticleDialog( {
			postId: 84808,
			postUrl: localizeUrl( 'https://wordpress.com/support/portfolios/' ),
		} );
};

const mapDispatchToProps = {
   activateModule,
   openTestimonialsSupport,
    openPortfoliosSupport,
};
```

I'd still recommend using https://wpcalypso.wordpress.com/devdocs/design/inline-support-link unless it wasn't a good fit here.

Reply:
Makes sense. Taking the hardcoded values out of the JSX, in other words? I'll refactor towards that as well.